stack-guideline.md

```diff
@@ -0,0 +1,68 @@
+# Abtion stack
+###### Or Ab-*stack*-tion!  🤦‍♂️
+![abtion image](https://media.giphy.com/media/ZVik7pBtu9dNS/giphy.gif)
+
+
+In Abtion we strive for the most efficient and easiest workflow.
+This docs is thought of being a docs we all can turn to if we get in doubt of how we deal with managing certain stuff, such as CI/CD.
+Also, to make it easier to on-board new employees.
+
+
+## Frontend
+As an Abtioneer we love React when working on the FE (Front-End), but we also care about Vue and is always curious of what exciting technologies that might pop-up in the future. But for now most of our FE apps is built with React.
+
+#### Testing
+React and [Jest](https://jestjs.io/) is a great match, which is also why we always test our React apps with Jest.
```

Comment:
is a great match, are a great match 🤔 
make a great match?
also: you can drop the also ;)

Reply:
```suggestion
React and [Jest](https://jestjs.io/) make a great match, which is why we always test our React apps with Jest.
```

Reply:
oh yea! suggestion feature!
would have saved some confusion 👍